src/main/java/krona/ui/Ui.java

```diff
@@ -0,0 +1,71 @@
+package krona.ui;
+
+import java.util.Scanner;
+
+import javafx.application.Platform;
+import javafx.stage.Stage;
+import krona.task.TaskList;
+
+
+/**
+ * The Ui class handles interactions with the user.
+ * It is responsible for displaying messages, receiving user input,
+ * and showing the list of tasks.
+ */
+public class Ui {
+    private Scanner scanner;
+    private String lastMessage;
+    private String combinedMessage;
+    private Stage stage;
+
+    /**
+     * Constructs a Ui object and initializes the scanner to read user input.
+     */
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public Ui(Stage stage) {
+        this.stage = stage;
+    }
+
+    public void goodbye() {
+        setCombinedMessage("Bye. Hope to see you again soon!");
+    }
+
+    public void exitApp() {
```

Comment:
Really like this implementation to exit from the javafx application after the user input bye! Good job!